receipt_layoutlm/receipt_layoutlm/trainer.py

```
@@ -1333,14 +1555,89 @@ def compute_metrics(eval_pred):
+                            return "O"
+                        if tag.startswith("B-") or tag.startswith("I-"):
+                            return tag[2:]
+                        return tag
+
+                    total_tokens = 0
+                    gold_entity_tokens = 0
+                    pred_entity_tokens = 0
+                    correct_entity_tokens = 0
+                    gold_o_tokens = 0
+                    correct_o_tokens = 0
+                    for true_seq, pred_seq in zip(y_true, y_pred):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)